tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{36,37,38}-test{,-alldeps,-devdeps}{,-cov}
-    py{36,37,38}-test-numpy{116,117,118}
-    py{36,37,38}-test-astropy{30,40,lts}
+    py{36,37,38,39,310,311}-test{,-alldeps,-devdeps}{,-cov}
```

Comment:
Apparently, it was a matter of giving the python version as a string instead of a float in the `ci_tests.yml` file (`3.10` was interpreted as a float representation of 3.1 🙄 )